Gemfile

```diff
@@ -1,18 +1,10 @@
-source "http://rubygems.org"
+# A sample Gemfile
+source "https://rubygems.org"
 
-if ENV.key?('PUPPET_VERSION')
```

Comment:
Not including puppet here also breaks the typical flow, eg.

```
bundle install
bundle exec rake spec
```

Will fail without puppet being present in the Gemfile.